### ISSUE TITLE
Nightly test test_miri_ifu_wcs fails due to missing WCS object in comparison file

### ISSUE THREAD

**Issue by jdavies-st** [STScI-JWST/jwst#2054]
The nightly regression test [test_miri_ifu_wcs](https://github.com/STScI-JWST/jwst/blob/master/jwst/tests_nightly/general/miri/test_miri_ifu_wcs.py) fails due to missing WCS object in comparison file.  The comparison file is located at
```
/data4/jwst_test_data/miri/test_wcs/ifu/jw00024001001_01101_00001_MIRIFUSHORT_assign_wcs.fits
```
This test was not (and is not) running on the Pandokia side.  Does this test need to be OK-ifed, fixed, or removed?  @nden?

I am setting the test to `xfail` for now.

**Comment by nden** [comment]
@jdavies-st I've updated the file in pandokia.

**Comment by jdavies-st** [comment]
Thanks!  I will close #2055.

**Comment by jdavies-st** [comment]
Resolved by #2065 and an update to the input file by @nden.

This test now passes as of [Build 44](https://boyle.stsci.edu:8081/job/RT/job/JWST/44/).  Closing issue.
